### DISCUSSION TITLE
Important touch features

### DISCUSSION THREAD

**Discussion by unBROken962** [ppy/osu#16296] [Ideas] [answered]
Make an area for streaming instead of x & z
(Touch input)

**Comment by bdach** [comment, marked as answer]
#4201

please search

**Reply by unBROken962** [reply]
About what?

**Reply by integerrr** [reply]
<img width="1062" alt="image" src="https://user-images.githubusercontent.com/7279624/147877344-310341aa-c9a8-4917-8861-7fbd712119a0.png">

**Reply by unBROken962** [reply]
Oh ok
